apps/organization/src/organization.service.ts

```diff
@@ -623,26 +623,38 @@ export class OrganizationService {
         userId
       );
 
-      let orgIds;
-      if (0 > getOrgs?.organizations?.length) {
-        orgIds = getOrgs?.organizations?.map(item => item.id);
+      if (0 === getOrgs?.organizations?.length) {
+        throw new NotFoundException(ResponseMessages.organisation.error.organizationNotFound);
       }
-       
-      const orgEcosystemDetails = await this._getOrgEcosystems(orgIds);
-      if (!orgEcosystemDetails || !Array.isArray(orgEcosystemDetails) || 0 === orgEcosystemDetails.length) {
-        throw new NotFoundException(ResponseMessages.ecosystem.error.ecosystemDetailsNotFound);
+
+      let orgIds;
+      let updatedOrgs;
+
+      if ('true' === process.env.IS_ECOSYSTEM_ENABLE) {
```

Comment:
Can we please add this with the appropriate value in the sample.env and demo.env and in our env trancking

Reply:
I have added this variable in sample.env and demo.env